public/index.svg

```diff
@@ -31,4 +31,8 @@
     <path d="M0 0h24v24H0z" fill="none"/>
     <path d="M7 11v2h10v-2H7zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/>
   </svg>
+  <svg id="done"  viewBox="0 0 512 512">
+   <path d="M0 0h24v24H0z" fill="none"/>
+  <path d="M256 8C119.033 8 8 119.033 8 256s111.033 248 248 248 248-111.033 248-248S392.967 8 256 8zm0 48c110.532 0 200 89.451 200 200 0 110.532-89.451 200-200 200-110.532 0-200-89.451-200-200 0-110.532 89.451-200 200-200m140.204 130.267l-22.536-22.718c-4.667-4.705-12.265-4.736-16.97-.068L215.346 303.697l-59.792-60.277c-4.667-4.705-12.265-4.736-16.97-.069l-22.719 22.536c-4.705 4.667-4.736 12.265-.068 16.971l90.781 91.516c4.667 4.705 12.265 4.736 16.97.068l172.589-171.204c4.704-4.668 4.734-12.266.067-16.971z"/>
```

Comment:
Indentation looks off here, have you heard of eslint? It will be super useful for things like this.

Reply:
@lukejacksonn should I change this, using eslint indentation?

Reply:
Okay , I'll leave that to you. Btw is there any other medium apart from git to communicate ,  like pm etc. If there is can you tell me ? I'd like to communicate with you there.